app/modules/ads/module.js

```diff
@@ -101,62 +103,76 @@ class Ads {
   }
 
   configureBillTheLizard(instantGlobals) {
```

Comment:
Imo BTL deserves for separate file/module. It's getting bigger.

Reply:
You right. Refactored to separate module.